cobigen-maven/cobigen-maven-systemtest/src/test/resources/testdata/systemtest/Oasp4JTemplateTest/TestExceptionOnNothingToGenerate/pom.xml

```diff
@@ -4,13 +4,11 @@
   <groupId>com.devonfw.cobigen</groupId>
   <artifactId>Oasp4JTemplateTest-TestClassLoadingWithTemplateFolder</artifactId>
   <version>dev-SNAPSHOT</version>
-
   <properties>
```

Comment:
As far as I know, we can know remove the complete `Oasp4JTemplateTest` folder as `devon4jTemplateTest` is testing the same, right?

Reply:
Yes, we can safely remove it I think.

Reply:
We can still keep them to make sure that the old structure works too.